core/src/main/java/org/apache/accumulo/core/fate/zookeeper/ZooUtil.java

```diff
@@ -72,7 +72,11 @@ public LockID(String root, String serializedLID) {
       if (lastSlash == 0) {
         path = root;
       } else {
-        path = root + "/" + sa[0].substring(0, lastSlash);
+        path = root;
+        if (!sa[0].startsWith("/")) {
+          path += "/";
+        }
```

Comment:
I had to add this because for some reason `sa[0]` starts with a `/` for the MANAGERS locks.